### PR TITLE
Feature/add citizenship countrycode

### PR DESCRIPTION
**Pull request by dd-sdang**
# Added country code to citizenship field

## What

- Updated the national registry api to return country code as well in the field for citizenship.
- Updated graphql queries that has the citizenship field
- Added lastUpdated selection to graphql query in application system
- Added logging in application system when querying national registry (I use console.log right now as the logging library doesnt work in frontend at the moment)

## Why

The country code is required in the application for health insurance

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/national-registry/src/lib/graphql/user.resolver.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class UserResolver {
     return this.nationalRegistryService.getUser(user.nationalId)
   }
 
-  @ResolveField('citizenship', () => String)
+  @ResolveField('citizenship', () => Citizenship)
   resolveCitizenship(@Parent() user: User): string {
     return user.citizenship.name
```
Comment (darrikonn) [inline libs/api/domains/national-registry/src/lib/graphql/user.resolver.ts:33]: Need to change the resolver as well. Shouldn't just return the name

**Review by darrikonn — CHANGES_REQUESTED** [review]
Otherwise looks good!

File: libs/api/domains/national-registry/src/lib/graphql/user.resolver.ts (hunk 1 above)
Comment (dd-sdang) [reply libs/api/domains/national-registry/src/lib/graphql/user.resolver.ts]: Got it! Updated :)

**Review by darrikonn — APPROVED** [review]
Nice

**Review by obmagnusson — APPROVED** [review]
LGTM

**Review by baering — APPROVED** [review]
LGTM 👍

**Pull request merged**